test/TestRunner.py

```diff
@@ -76,12 +76,17 @@ def test_runner_exclude_paths(default_rules_collection: RulesCollection) -> None
     assert len(matches) == 0
 
 
-def test_runner_exclude_globs(default_rules_collection: RulesCollection) -> None:
+@pytest.mark.parametrize(
+    ('exclude_path'), ('**/example.yml', 'examples/**/example.yml')
```

Comment:
Those samples are not the best since they, while being valid globs, are too similar to the extended (wildcard) matches that are supported by the wcmatch library for example. Maybe we should add some more bash-like expressions here? Things like `*.yml`, etc.